app/helpers/codelist_helper.rb

```diff
@@ -34,6 +34,11 @@ def region_select_options
     objects.unshift(OpenStruct.new(name: "Developing countries, unspecified", code: "998")).uniq
   end
 
+  def country_select_options
+    objects = yaml_to_objects(entity: "activity", type: "recipient_country", with_empty_item: false)
```

Comment:
Should this be sorted?

Reply:
Yes, I think so:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/ce530674dc8d497b0d1876852d8a3f7e711349f7/app/helpers/codelist_helper.rb#L11

Reply:
Ah. Great.